readme.md

```diff
@@ -73,6 +73,12 @@ Values: `start` `middle` `end`
 
 Position to truncate the string.
 
+##### space
+
+Type: `boolean`<br>
+Values: `true` `false`
+
+Add space to ellipsis. Depends on the `position` option.
```

Comment:
Add a code example here of how it looks with the different `position` options. It's not immediate clear to the user how it would look.

Reply:
I have added them to the main example.